src/components/Axis/Ticks.tsx

```diff
@@ -25,6 +27,7 @@ export interface TicksProps extends Omit<TickProps, 'line' | 'text'> {
   ticks: number[];
   scale: ScaleLinear<number, number, never>;
   scientific?: boolean;
+  secondaryTicksHidden?: boolean;
```

Comment:
Because the name of the main feature is `hiddenTicks`, what do you think of naming the variable like this?
```suggestion
  hiddenSecondaryTicks?: boolean;
```